incubator/cockroachdb/README.md

```diff
@@ -0,0 +1,184 @@
+(1234, 10000.50);
+INSERT 1
+root@my-release-cockroachdb-public:26257> SELECT * FROM bank.accounts;
++------+---------+
+|  id  | balance |
++------+---------+
+| 1234 | 10000.5 |
++------+---------+
+(1 row)
+root@my-release-cockroachdb-public:26257> \q
+Waiting for pod default/cockroach-client to terminate, status is Running
+pod "cockroach-client" deleted
+```
+
+## Cluster health
+
+Because our pod spec includes regular health checks of the CockroachDB processes,
+simply running `kubectl get pods` and looking at the `STATUS` column is sufficient
+to determine the health of each instance in the cluster.
+
+If you want more detailed information about the cluster, the best place to look
+is the admin UI.
+
+## Accessing the admin UI
+
+If you want to see information about how the cluster is doing, you can try
+pulling up the CockroachDB admin UI by port-forwarding from your local machine
+to one of the pods (replacing "release-cockroachdb-0" with one of your pods' names):
+
+```shell
+kubectl port-forward release-cockroachdb-0 8080
+```
+
+Once you’ve done that, you should be able to access the admin UI by visiting
+http://localhost:8080/ in your web browser.
+
+## Failover
+
+If any CockroachDB member fails it gets restarted or recreated automatically by
+the Kubernetes infrastructure, and will rejoin the cluster automatically when
+it comes back up. You can test this scenario by killing any of the pets:
+
+```shell
+kubectl delete pod my-release-cockroachdb-1
+```
+
+```shell
+$ kubectl get pods -l "component=my-release-cockroachdb"
+NAME                      READY     STATUS        RESTARTS   AGE
+my-release-cockroachdb-0  1/1       Running       0          5m
+my-release-cockroachdb-2  1/1       Running       0          5m
+```
+
+After a while:
+
+```console
+$ kubectl get pods -l "component=my-release-cockroachdb"
+NAME                      READY     STATUS        RESTARTS   AGE
+my-release-cockroachdb-0  1/1       Running       0          5m
+my-release-cockroachdb-1  1/1       Running       0          20s
+my-release-cockroachdb-2  1/1       Running       0          5m
+```
+
+You can check state of re-joining from the new pod's logs:
+
+```console
+$ kubectl logs my-release-cockroachdb-1
+[...]
+I161028 19:32:09.754026 1 server/node.go:586  [n1] node connected via gossip and
+verified as part of cluster {"35ecbc27-3f67-4e7d-9b8f-27c31aae17d6"}
+[...]
+cockroachdb-0.my-release-cockroachdb.default.svc.cluster.local:26257
+build:      beta-20161027-55-gd2d3c7f @ 2016/10/28 19:27:25 (go1.7.3)
+admin:      http://0.0.0.0:8080
+sql:
+postgresql://root@my-release-cockroachdb-1.my-release-cockroachdb.default.svc.cluster.local:26257?sslmode=disable
+logs:       cockroach-data/logs
+store[0]:   path=cockroach-data
+status:     restarted pre-existing node
+clusterID:  {35ecbc27-3f67-4e7d-9b8f-27c31aae17d6}
+nodeID:     2
+[...]
+```
+
+## Scaling
+
+Scaling should typically be managed via the `helm upgrade` command, but if
```

Comment:
It does not work yet, good catch. It just gives the error:

```
Error: UPGRADE FAILED: PetSet.apps "gangly-ant-cockroachdb" is invalid: spec: Forbidden: updates to petset spec for fields other than 'replicas' are forbidden.
```

Added a little blurb about using kubectl patch.